src/plugins/data_views/server/fetcher/lib/jobs_compatibility.ts

```diff
@@ -66,16 +68,18 @@ export function mergeJobConfigurations(jobs = []) {
         }
         // If aggregation already exists, attempt to merge it
         else {
-          const fieldAgg = allAggs[aggName][fieldName];
-
+          const fieldAgg = allAggs[aggName][fieldName] as object | null;
           switch (aggName) {
             // For histograms, calculate the least common multiple between the
             // new interval and existing interval
             case 'histogram':
-              // TODO: Fix this with LCD algorithm
-              const intervals = [fieldAgg.interval, agg.interval].sort((a, b) => a - b);
-              const isMultiple = intervals[1] % intervals[0] === 0;
-              fieldAgg.interval = isMultiple ? intervals[1] : intervals[0] * intervals[1];
+              if (fieldAgg) {
+                const aggInterval = (agg as any).interval ?? agg.calendar_interval; // FIXME the interface infers only that calendar_interval property is valid
```

Comment:
I couldn't find an `interval` field on the interface that is expected here, I thought maybe it should be `calendar_interval`